### PR TITLE
docs/source: added documentation for trivy

### LINKED ISSUE #2: Need to document learning path for trivy
If we want to contribute to trivy later, we need a place to gather the links pointing to the "must read" data, like architecture diagram, deployment guides, contribution guides.

### PR DESCRIPTION
**Pull request by mssola**
Fixes #2

Signed-off-by: Miquel Sabaté Solà <msabate@suse.com>

### CODE REVIEW

**Comment by evrardjp** [comment]
Awesome! I will try this!

File: docs/source/trivy.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#####
+trivy
+#####
+
```
Comment (evrardjp) [inline docs/source/trivy.rst:4]: Can you give a few words of what trivy is, and how does it fit the ecosystem with kube-hunter, etc?

**Review by evrardjp — CHANGES_REQUESTED** [review]
I think it would be best if we also include how to get onboarded, and what the project is useful for.

Nevertheless, it sets the bar quite high on what to document for other projects.

File: docs/source/trivy.rst (hunk 1 above)
Comment (evrardjp) [reply docs/source/trivy.rst:4]: I would also mention two other things:
- it's in go
- how to get started (links to onboarding).

Comment (evrardjp) [reply docs/source/trivy.rst:4]: I would also mention the fact there is a github action for it :)

Comment (mssola) [reply docs/source/trivy.rst:4]: > It's in go

I don't think it's really relevant . Plus, you can take a look at this fact by checking the provided links or the OBS package.

> how to get started (links to onboarding).

Will do

> I would also mention the fact there is a github action for it :)

Already documented

Comment (mssola) [reply docs/source/trivy.rst:4]: > I don't think it's really relevant . Plus, you can take a look at this fact by checking the provided links or the OBS package.

Added it anyways :stuck_out_tongue_winking_eye:

**Comment by mssola** [comment]
@evrardjp review again

**Review by evrardjp — APPROVED** [review]
Very good entry level info!

**Pull request merged**